_data/events/London-Pivotal.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "GDCR 17 @ Pivotal London",
+  "url": "https://www.eventbrite.com/e/global-day-of-code-retreat-pivotal-tickets-37119002937",
+  "moderators": [
+    "@deniseyu21",
+		"jamesjoshuahill",
+ 		"chewymeister"
+  ],
+  "location": {
+    "city": "London",
+    "country": "UK",
```

Comment:
I guess it's going to happen at Pivotal, right? If you add the `location.coordinates` properties here the event will show up on the map as well :)